Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
This fix is orthogonal to streaming support as:
- if it was supported - it continues to support it
- if it wasn't supported - it continues to not support it

but I really think that streaming was never supported in RxAlamofire